src/md_lammps.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include <core/md_str.h>
+
+//All lammps units should have a 1:1 mapping to the md_molecule according to https://docs.lammps.org/2001/units.html
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct md_allocator_i;
+struct md_molecule_t;
+struct md_molecule_loader_i;
+
+//Contains potential data format identifiers
+typedef enum lammps_atom_data_format {
+	ATOM_IDX,
+	MOL_IDX,
+	ATOM_TYPE,
+	PARTIAL_CHARGE,
+	ATOM_X,
+	ATOM_Y,
+	ATOM_Z,
+	NX,
+	NY,
+	NZ,
+	res_name,
```

Comment:
Why is res_name and res_id in smaller case?

Reply:
Adjusted them to be CAPS